### PR TITLE
Search Bar Finished

### PR DESCRIPTION
**Pull request by dhyum9**

### CODE REVIEW

File: react-app/src/components/SearchBar/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              {searchType === 'restaurants' && (
+                <div className='search-bar-result' onClick={() => handleClick(data.id)}>
+                  <div className='search-bar-result-img-container'>
+                    <img className='search-bar-result-img' src={data.image_url}></img>
+                  </div>
+                  <div className='search-bar-result-info-container'>
+                    <div>
+                      {data.name} ({data.address})
+                    </div>
+                    <div>{templatePrice.join("")} &#183; {data.type}</div>
```
Comment (kwongit) [inline react-app/src/components/SearchBar/index.js:71]: what's this &#183 syntax?

Comment (dhyum9) [reply react-app/src/components/SearchBar/index.js:71]: It's called an HTML Entity, basically renders in a middle dot symbol

File: react-app/src/components/Navigation/index.js [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
 	const sessionUser = useSelector(state => state.session.user);
+	const { restaurantId } = useParams();
+  const getRestaurants = useSelector((state) => state.restaurant.allRestaurants);
+	const getMenuItems = useSelector((state) => state.menuItems.allMenuItems);
+	const menuItems = Object.values(getMenuItems);
+  const restaurants = Object.values(getRestaurants);
+
+  useEffect(() => {
+    dispatch(thunkGetRestaurants());
+		dispatch(thunkGetMenuItems(restaurantId));
```
[the remark this answers is not in the document]
Comment (dhyum9) [reply react-app/src/components/Navigation/index.js:23]: think it's just Github, the indentation is consistent on VSCode!

File: react-app/src/components/SearchBar/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import { useHistory } from 'react-router';
+
+const SearchBar = ({ placeholder, data, searchType}) => {
+const history = useHistory();
+const [filteredData, setFilteredData] = useState([]);
+const [wordEntered, setWordEntered] = useState("");
+
+const handleClick = (id) => {
+  history.push(`/restaurants/${id}`);
+  setFilteredData([]);
```
Comment (kwongit) [inline react-app/src/components/SearchBar/index.js:12]: you could call clearInput() here

**Review by kwongit — APPROVED** [review]

**Pull request merged**